lib/Doctrine/ORM/Persisters/Entity/BasicEntityPersister.php

```diff
@@ -93,7 +93,6 @@ class BasicEntityPersister implements EntityPersister
 {
     /** @var string[] */
     private static $comparisonMap = [
-        Comparison::EQ          => '= %s',
```

Comment:
Why is this being removed? Eq/Is being the same?

Reply:
Yeap, here's the PHPStan error:

```
 ------ ----------------------------------------------------------------------------------------------
  Line   Doctrine/ORM/Persisters/Entity/BasicEntityPersister.php
 ------ ----------------------------------------------------------------------------------------------
  96     Array has 2 duplicate keys with value '=' (\Doctrine\Common\Collections\Expr\Comparison::EQ,
         \Doctrine\Common\Collections\Expr\Comparison::IS).
 ------ ----------------------------------------------------------------------------------------------
```